MyHeartCounts/Account/Demographics/DemographicsAccountKeys.swift

```diff
@@ -152,6 +152,17 @@ extension AccountDetails {
     
     @AccountKey(id: "futureStudies", name: "", category: .demographics, options: .mutable, as: Bool.self)
     var futureStudies: Bool?
+    
+    
+//    @AccountKey(
+//        id: "dateOfEnrollment",
+//        name: "Date of Enrollment",
+//        category: .other,
+//        options: .mutable,
+//        as: Date.self,
+//        initial: .empty(.distantPast),
+//    )
+//    public var dateOfEnrollment: Date?
```

Comment:
TODO: enable all of this once the `dateOfEnrollment` in firebase has been updated to a timestamp rather than a string (cc @PaulGoldschmidt)